service-front/module/Application/src/Services/OpgApiService.php

```diff
@@ -82,7 +82,7 @@ public function checkNinoValidity(string $nino): bool
             return false;
         }
 
-        return $this->responseData['status'] === 'valid';
+        return $this->responseData['status'] === 'NINO check complete';
```

Comment:
Again something for a future ticket, but the API should be returning codes (e.g. "PASS") rather than English langauge strings

Reply:
Curious what you mean here @gregtyler I was thinking of HTTP codes but is there some alternative here because a PASS is still a string? Or is it just that its one short string rather than a sentence?

Reply:
Created new ticket for this

Reply:
@fazemiah-moj The latter. The API should return things like this in a generic standardised form (e.g. `[A-Z0-9_]`) and the frontend will be responsible for any translations into human-readable sentences. It makes it easier to change content later, and would make it easier to support other languages (i.e. Welsh) in the future